connect_transformations/lookup_ff_request/mixins.py

```diff
@@ -108,24 +114,26 @@ async def get_request(self, lookup):
         except KeyError:
             pass
 
-        for attempts_left in range(MAX_API_CALL_CONNECTION_ERROR_RETRIES, -1, -1):
-            try:
-                result = await self.retrieve_ff_requests(lookup)
-                break
-            except ClientError:
-                if not attempts_left:
-                    raise
-                continue
+        result = await self.retrieve_ff_requests(lookup)
```

Comment:
Can you please clarify why you decided to remove retries?

Reply:
This timeout was needed because the Connect client wasn't retrying in case of connection errors.

It was fixed https://github.com/cloudblue/connect-python-openapi-client/pull/74

What I forgot to do is bump the requirements to apply this patch.

This retry should be also deleted in subscriptions lookup. I will do it a bit later.